api/proto/info/info.proto

```diff
@@ -631,6 +634,26 @@ enum MaintenanceModeReason {
   MAINTENANCE_MODE_REASON_LOW_DISK_SPACE = 3;
 }
 
+// Information about attestation process
+message AttestationInfo {
+  AttestationState state = 1;   //current state of attestation
+  ErrorInfo error = 2;          //last error
+}
+
+// Different states of attestation process
+// Must match the values in pkg/pillar/attest.State
+enum AttestationState {
+  ATTESTATION_STATE_NONE = 0;                    //State when (Re)Starting attestation
+  ATTESTATION_STATE_NONCE_WAIT = 1;              //Waiting for response from Controller for Nonce request
+  ATTESTATION_STATE_INTERNAL_QUOTE_WAIT = 2;     //Waiting for internal PCR quote to be published
+  ATTESTATION_STATE_INTERNAL_ESCROW_WAIT = 3;    //Waiting for internal Escrow data to be published
+  ATTESTATION_STATE_ATTEST_WAIT = 4;             //Waiting for response from Controller for PCR quote
+  ATTESTATION_STATE_ATTEST_ESCROW_WAIT = 5;      //Waiting for response from Controller for Escrow data
+  ATTESTATION_STATE_RESTART_WAIT = 6;            //Waiting for restart timer to expire, to start all over again
+  ATTESTATION_STATE_COMPLETE = 7;                //Everything w.r.t attestation is complete
+  ATTESTATION_STATE_ANY = 8;                     //Not a real state, placeholder for compatibility with future modifications
```

Comment:
Why do you need this at the end?

Reply:
I put it here to have no problems with future state additions, but correct, having this state now no make sense. I removed that state and put a comment on top of enum.